spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/sqlserver/V4_1__Task_Sequence.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.db.migration.sqlserver;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.flywaydb.core.api.migration.Context;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.cloud.dataflow.common.flyway.AbstractMigration;
+import org.springframework.cloud.dataflow.common.flyway.SqlCommand;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.SingleConnectionDataSource;
+
+/**
+ * Migration ensuring that {@code TASK_SEQ} sequence exists.
+ * Done for {@code mssql} via java and suppressing error as it doesn't support
+ * "create sequence if not exists". Also we migrate from {@code TASK_SEQ} table
+ * into sequence by taking out a correct seq value.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+public class V4_1__Task_Sequence extends AbstractMigration {
+
+	private static final Logger logger = LoggerFactory.getLogger(R__Hibernate_Sequence.class);
+
+	// Caused by: org.springframework.jdbc.UncategorizedSQLException:
```

Comment:
Do we need this comment?

Reply:
I was a reference why things are done like this. Could remove thought and fix wrong logger class.